### DISCUSSION TITLE
Welcome to hoylabs!

### DISCUSSION THREAD

**Discussion by schlimmchen** [hoylabs/OpenDTU-OnBattery#1313] [Announcements]
Deutsch weiter unten.

## English Announcement

`hoylabs` is the name of the newly founded GitHub organisation, which now hosts `OpenDTU-OnBattery` and `OpenDTU-OnBattery-docs`.

These repositories/projects were transferred to this organisation by their respective previous owners, including issues, discussions, the wiki, etc. Links to the old namespaces `helgeerbe/OpenDTU-OnBattery` and `schlimmchen/OpenDTU-OnBattery-docs` will redirect here.

@AndreasBoehm and I founded the organisation. @helgeerbe is the third member. Going forward, we will maintain OpenDTU-OnBattery together.

Existing forked repositories now appear forked from `hoylabs/OpenDTU-OnBattery`. No action is required by owners of forks.

## Ankündigung auf Deutsch

`hoylabs` ist der Name der neu gegründeten GitHub-Organisation, die nun `OpenDTU-OnBattery` und `OpenDTU-OnBattery-docs` hostet.

Diese Repositories/Projekte wurden von ihren jeweiligen Vorbesitzern in diese Organisation übertragen, einschließlich Issues, Diskussionen, Wiki usw. Links zu den alten Namespaces `helgeerbe/OpenDTU-OnBattery` und `schlimmchen/OpenDTU-OnBattery-docs` leiten jetzt hierhin weiter.

@AndreasBoehm und ich haben die Organisation gegründet. @helgeerbe ist das dritte Mitglied. Zukünftig werden wir OpenDTU-OnBattery gemeinsam pflegen.

Vorhandene geforkte Repositories erscheinen jetzt als Fork von hoylabs/OpenDTU-OnBattery. Für die Besitzer der Forks ist keine Aktion erforderlich.

**Comment by tutti83** [comment]
Hallo, ich würde mich sehr freuen wenn man auch einen Trucki T2SG (https://trucki.de/t2sg/) einfügen könnte !!!

viele dank 
Lukas

**Reply by AndreasBoehm** [reply]
Das sollte mit der T2H/G integration funktionieren ohne Änderungen an der Firmware 👍

**Reply by AndreasBoehm** [reply]
EDIT: sorry, das wir nicht klappen. Das ist ja schon eine eigene Steuerung.

**Comment by tutti83** [comment]
Ich habe einen T2HG schon laufen! Ladegerät und wollte den Inverter auch integrieren!
